### DISCUSSION TITLE
Python requests can not be used

### DISCUSSION THREAD

**Discussion by 3699999** [thebaselab/codeapp#1053] [Q&A] [answered]
For my word the request is important but as I know that the requests can not be used may this is a bug?

**Comment by bummoblizard** [comment, marked as answer]
Can you show the error message?

**Reply by 3699999** [reply]
puzzle_game $ javac
/var/mobile/Containers/Data/Application/DE0C909B-4172-49DC-A98D-3AB2200F3D0/Documents/puzzle_game/src/com/yzy/ui/App.java:10: ## i#
：找不到符号
new GameJFrame () ;
符号：类 GameJFrame
位置：类 App
1个錯淏
错误：找不到或无法加载主类 App
puzzle_game $ O

**Reply by 3699999** [reply]
jframe…… can't be used

**Reply by bummoblizard** [reply]
This is expected. Code does not support Java GUI because of they are designed to be used on a desktop environment. Python's requests should be available though.

**Reply by 3699999** [reply]
thanks a lot
